sample-code/javascript-webdriverio/test/basic/android-create-web-session.test.js

```diff
@@ -1,29 +1,23 @@
 const webdriverio = require("webdriverio");
 const androidOptions = require("../../helpers/caps").androidWebOptions;
-const app = require("../../helpers/apps").androidApiDemos;
 const assert = require("chai").assert;
 
-describe("Create Chrome web session", function() {
+describe("Create Chrome web session", () => {
   let client;
 
-  before(function() {
-    client = webdriverio.remote(androidOptions);
-    return client.init();
+  before(async() => {
+    client = await webdriverio.remote(androidOptions);
   });
 
-  after(function() {
-    return client.end();
+  after(async() => {
```

Comment:
Do not use fat arrow functions within mocha apparatus (see https://mochajs.org/#arrow-functions).